service-pdf/docker/app/Dockerfile

```
@@ -1,5 +1,7 @@
 FROM composer:2.8.12@sha256:5248900ab8b5f7f880c2d62180e40960cd87f60149ec9a1abfd62ac72a02577c AS composer
 
+FROM pdftk/pdftk:edge@sha256:c949f919fd5309ac592008619f873a248f4e06a5f25b284821ec2d59857e17c0 AS pdftk
```

can we pin this to a semver tag instead of edge for stability? Is the maintainer keeping up with releasing tags?

3.3.3 is available, and a month old https://hub.docker.com/layers/pdftk/pdftk/3.3.3/images/sha256-00cc180502216fb4f75a097c1c25464fc24842b8d4555e059e0eaad340199ad8


```suggestion
FROM pdftk/pdftk:3.3.3@sha256:00cc180502216fb4f75a097c1c25464fc24842b8d4555e059e0eaad340199ad8 AS pdftk
```